doc/docs/master/how-tos/mount-volume.md

```diff
@@ -0,0 +1,159 @@
+# Mount a Volume in a Pipeline
+
+You may have a local folder that you want your pipeline to write
+files to. You can mount that folder as a volume in Kubernetes
```

Comment:
There's a lot more use cases than mounting local folders, e.g. mounting networked volumes like NFS. [Here's the full list](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes).

Reply:
we don't need to list them all but I've added a clarification. Also, they would have to write the output to that mount instead of `pfs/`, right?